PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java

```diff
@@ -171,41 +186,53 @@ public void onAttach(Activity activity) {
     @Override
     public void onResume() {
 
-        //Get the search parameters
+        // Get the search parameters
         SharedPreferences searchPreferences = getActivity().getSharedPreferences(SearchFragment.SEARCH_PARAMETERS, 0);
```

Comment:
do you think using shared preferences is the best option for passing "first name" and "last name"?
if you look in SearchFragment where it creates this shared preferences, you'll see a comment that says "//Save parameters to SharedPrefs to access from the next activity." 
but we never actually use it in different activities - only in CheckinActivity.